my-app/src/shared/components/InputComponent.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { baseUrl } from "../BaseUrl";
+import { Key } from "../Key";
+import { makeObservable, observable, action } from "mobx";
+
+export class InputComponent extends React.Component {
```

Comment:
you can remove this file for now, and move request logic for this file to the parent component (App in our case)